bin/run-l10n-extraction

```diff
@@ -53,13 +59,14 @@ run_l10n_extraction() {
   info "Creating and switching to branch '$branch'"
   git checkout -b "$branch"
 
-  NODE_APP_INSTANCE="$app" bin/extract-locales
+  info "Extracting locales (this can take several minutes)"
+  NODE_APP_INSTANCE="$app" bin/extract-locales >> "$LOG_FILE" 2>&1
 
   local git_diff_stat
   git_diff_stat=$(git diff --shortstat)
 
   if [[ -z "$git_diff_stat" ]] || [[ "$git_diff_stat" == *"$DIFF_WITH_ONE_LINE_CHANGE"* ]]; then
-    info "No changes, aborting"
+    info "No locale changes for $app, aborting"
```

Comment:
To be *even more* explicit, how about: "No locale changes for $app, nothing to update, ending process for $app"

I think part of the confusion around this message was the word "aborting" and the thought that it meant something went wrong.